src/features/import/hooks/useColumn.ts

```diff
@@ -60,14 +67,25 @@ export default function useColumn(orgId: number) {
       label: globalMessages.personFields[fieldSlug](),
       value: `field:${fieldSlug}`,
     }));
-  const customFieldsOptions: Option[] = customFields.map((field) => ({
-    label: field.title,
-    value: `field:${field.slug}`,
-  }));
+
+  const customFieldsOptions: Option[] = customFields.map((field) => {
+    if (field.type === CUSTOM_FIELD_TYPE.DATE) {
+      return {
+        label: field.title,
+        value: `date:${field.slug}`,
+      };
```

Comment:
This deviates a little bit from the approach that would feel most natural to me, which is to still consider this a `field` but then handle that field differently elsewhere based on the type of the field. But I haven't tried it so I don't know if that causes problems as it cascades, so I am not suggesting that you change it now.

Reply:
The reason for it is that it is how I decide what column kind to create... I think you are right, that it would make more sense to use `field`, but I could not figure out another way to know how to decide what type of data was being imported... 

in `FieldSelect.tsx`
![bild](https://github.com/zetkin/app.zetkin.org/assets/58265097/f0c7cd77-8899-49d8-9e82-b6403f287edd)